plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,382 @@
+package whatap
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+
+	//"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	NETSRC_AGENT_ONEWAY = 10
+	VALUE_DECIMAL       = 20
+	VALUE_DOUBLE        = 40
+	VALUE_TEXT          = 50
+	VALUE_MAP           = 80
```

Comment:
I think this should be moved to the `DataOutputX` location and the write functions should directly write that type.

Reply:
Moved to a library outpside of telegraf